src/other/Legend.js

```diff
@@ -82,6 +82,7 @@
         Table.prototype.enter.apply(this, arguments);
         this.renderHtmlDataCells(true);
         this.fixedHeader(false);
+        this.fixedSize(true);
```

Comment:
The legend and mega chart need some fixing and I suspect any fixes there will affect this as well.